public/index.html

```diff
@@ -1,37 +1,290 @@
+                    <svg role="img" height="16" width="16" viewBox="0 0 16 16">
+                        <path d="M3.3 1a.7.7 0 01.7.7v5.15l9.95-5.744a.7.7 0 011.05.606v12.575a.7.7 0 01-1.05.607L4 9.149V14.3a.7.7 0 01-.7.7H1.7a.7.7 0 01-.7-.7V1.7a.7.7 0 01.7-.7h1.6z">
+                        </path>
+                    </svg>
+                </button>
+                         
+                <button type="button" class="player-controls__play" aria-label="Слушать">
+                    <svg  role="img" height="26" width="26" viewBox="0 0 16 16">
+                        <path d="M3 1.713a.7.7 0 011.05-.607l10.89 6.288a.7.7 0 010 1.212L4.05 14.894A.7.7 0 013 14.288V1.713z">
+                        </path>
+                    </svg>
+                </button>
+                <button type="button" class="player-controls__button" aria-label="Далее">
+                    <svg role="img" height="16" width="16" viewBox="0 0 16 16">
+                        <path d="M12.7 1a.7.7 0 00-.7.7v5.15L2.05 1.107A.7.7 0 001 1.712v12.575a.7.7 0 001.05.607L12 9.149V14.3a.7.7 0 00.7.7h1.6a.7.7 0 00.7-.7V1.7a.7.7 0 00-.7-.7h-1.6z">
+                        </path>
+                    </svg>
+                </button>
+                <button type="button" class="player-controls__button" aria-label="Повторять">
+                    <svg role="img" height="16" width="16" viewBox="0 0 16 16">
+                        <path d="M0 4.75A3.75 3.75 0 013.75 1h8.5A3.75 3.75 0 0116 4.75v5a3.75 3.75 0 01-3.75 3.75H9.81l1.018 1.018a.75.75 0 11-1.06 1.06L6.939 12.75l2.829-2.828a.75.75 0 111.06 1.06L9.811 12h2.439a2.25 2.25 0 002.25-2.25v-5a2.25 2.25 0 00-2.25-2.25h-8.5A2.25 2.25 0 001.5 4.75v5A2.25 2.25 0 003.75 12H5v1.5H3.75A3.75 3.75 0 010 9.75v-5z">
+                        </path>
+                    </svg>
+                </button>
+            </div>
+
+            <div class="volume-bar" dir="ltr">
+                <button class="volume-bar__icon-button player-controls__button" aria-label="Выключить звук" aria-describedby="volume-icon" aria-expanded="false">
+                    <svg role="presentation" height="16" width="16" aria-label="Высокая громкость" id="volume-icon" viewBox="0 0 16 16" class="Svg-sc-1bi12j5-0 hDgDGI">
+                        <path d="M9.741.85a.75.75 0 01.375.65v13a.75.75 0 01-1.125.65l-6.925-4a3.642 3.642 0 01-1.33-4.967 3.639 3.639 0 011.33-1.332l6.925-4a.75.75 0 01.75 0zm-6.924 5.3a2.139 2.139 0 000 3.7l5.8 3.35V2.8l-5.8 3.35zm8.683 4.29V5.56a2.75 2.75 0 010 4.88z">
+                        </path>
+                        <path d="M11.5 13.614a5.752 5.752 0 000-11.228v1.55a4.252 4.252 0 010 8.127v1.55z">
+                        </path>
+                    </svg>
+                </button>
+                <div class="p1ULRzPc4bD8eQ4T_wyp DFtdzavKSbEhwKYkPTa6">
+                    <label class="hidden-visually">
+                        <input type="range" min="0" max="1" step="0.1" value="0.9354838709677419">
```

Comment:
value можно округлить до:
```suggestion
                        <input type="range" min="0" max="1" step="0.1" value="0.9">
```